go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/cenkalti/rpc2 v0.0.0-20140912135055-44d0d95e4f52 // indirect
 	github.com/contiv/libOpenflow v0.0.0-20200728044739-7c6534390721
 	github.com/contiv/libovsdb v0.0.0-20170227191248-d0061a53e358
+	github.com/google/martian v2.1.0+incompatible
```

Comment:
I think the project uses "github.com/sirupsen/logrus"

Reply:
Thanks, I didn't realize that the auto-import used this one.